src/explore-education-statistics-admin/src/services/methodology/methodologyService.ts

```diff
@@ -91,6 +104,62 @@ const service = {
   ): Promise<BasicMethodology> {
     return client.put(`/methodology/${methodologyId}/status`, updateRequest);
   },
+
+  addContentSectionBlock(
+    methodologyId: string,
+    sectionId: string,
+    block: ContentBlockPostModel,
+    isAnnexes = false,
+  ): Promise<EditableContentBlock> {
+    return client.post<EditableContentBlock>(
+      `/methodology/${methodologyId}/content/section/${sectionId}/blocks/add${
+        isAnnexes ? '?type=annexes' : ''
```

Comment:
Does the `type` param accept `content`?

Reply:
AFAIK it if the type param is not provided, then it defaults to `content` and I think the only type param that works is annexes...